### PR TITLE
Update taxonomy term markup for Neue 6.0.

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes
- Updates markup on taxonomy term page for Neue 6.0.

For review: @DoSomething/front-end

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/templates/taxonomy/taxonomy-term.tpl.php [hunk 1]
```diff
@@ -13,71 +13,80 @@
...
           </ul>
-        </section>
-      <?php endif; ?>
 
-      <?php if (isset($global_copy['campaign_value_proposition'])): ?>
-        <h2 class="inline--alt-color"><?php print t('What You Get'); ?></h2>
-        <?php print $global_copy['campaign_value_proposition']; ?>
-      <?php endif; ?>
+          <section class="sources">
+            <h3 class="__title js-toggle-sources"><?php print t('Sources'); ?></h3>
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/templates/taxonomy/taxonomy-term.tpl.php:64]: Curious, are we doing the heading classes stuff?

File: lib/themes/dosomething/paraneue_dosomething/templates/taxonomy/taxonomy-term.tpl.php [hunk 2]
```diff
@@ -13,71 +13,80 @@
...
+
+          <?php print $intro; ?>
+          <?php if (isset($modals)): ?>
+            <?php print $modals; ?>
+          <?php endif; ?>
+        </div>
+        <?php if (isset($intro_image) || isset($intro_video)): ?>
+        <div class="container__body -half">
+          <?php if (isset($intro_video)): ?>
+            <div class="media-video">
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/templates/taxonomy/taxonomy-term.tpl.php:31]: Future update, now with the `media` class maybe this changes to `media -video`? Not sure.

**Comment by weerd** [comment]
Couple questions which don't need answers now, but looks good overall :+1:

File: lib/themes/dosomething/paraneue_dosomething/templates/taxonomy/taxonomy-term.tpl.php (hunk 1 above)
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/taxonomy/taxonomy-term.tpl.php:64]: We definitely could use that here! I haven't gotten around to using it much yet.

File: lib/themes/dosomething/paraneue_dosomething/templates/taxonomy/taxonomy-term.tpl.php (hunk 2 above)
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/taxonomy/taxonomy-term.tpl.php:31]: +1, confusing naming. Worth considering a fix.

**Pull request merged**